internal/app/config.go

```diff
@@ -78,6 +79,31 @@ func ValidateClusterJSONDataConfig(config Config) error {
 	return nil
 }
 
+func ValidateClusterOfflineReviewConfig(config Config) error {
+	if len(config.Clusters) < 1 {
```

Comment:
i'm wondering if that check is needed. If I specify dumpFile in a config, are the clusters still required ?

Reply:
agree